--- a/libavfilter/blend.h	2026-05-27 20:44:30
+++ b/libavfilter/blend.h	2026-05-27 20:44:44
@@ -66,6 +66,7 @@
     BLEND_STAIN,
     BLEND_INTERPOLATE,
     BLEND_HARDOVERLAY,
+    BLEND_INCLUSION,
     BLEND_NB
 };
 
--- a/libavfilter/blend_modes.c	2026-05-27 20:44:30
+++ b/libavfilter/blend_modes.c	2026-05-27 20:45:05
@@ -130,6 +130,7 @@
 fn(burn,       BURN(A, B))
 fn(softlight,  CLIP(A * A / MAX + (2 * (B * ((A * (MAX - A)) / MAX) / MAX))))
 fn(exclusion,  A + B - 2 * A * B / MAX)
+fn(inclusion,  MAX - A - B + 2 * A * B / MAX)
 fn(pinlight,   (B < HALF) ? FFMIN(A, 2 * B) : FFMAX(A, 2 * (B - HALF)))
 fn(phoenix,    FFMIN(A, B) - FFMAX(A, B) + MAX)
 fn(reflect,    (B == MAX) ? B : FFMIN(MAX, (A * A / (MAX - B))))
--- a/libavfilter/vf_blend.c	2026-05-27 20:44:30
+++ b/libavfilter/vf_blend.c	2026-05-27 20:44:47
@@ -85,6 +85,7 @@
     { "divide",     "", 0, AV_OPT_TYPE_CONST, {.i64=BLEND_DIVIDE},     0, 0, FLAGS, .unit = "mode" },
     { "dodge",      "", 0, AV_OPT_TYPE_CONST, {.i64=BLEND_DODGE},      0, 0, FLAGS, .unit = "mode" },
     { "exclusion",  "", 0, AV_OPT_TYPE_CONST, {.i64=BLEND_EXCLUSION},  0, 0, FLAGS, .unit = "mode" },
+    { "inclusion",  "", 0, AV_OPT_TYPE_CONST, {.i64=BLEND_INCLUSION},  0, 0, FLAGS, .unit = "mode" },
     { "extremity",  "", 0, AV_OPT_TYPE_CONST, {.i64=BLEND_EXTREMITY},  0, 0, FLAGS, .unit = "mode" },
     { "freeze",     "", 0, AV_OPT_TYPE_CONST, {.i64=BLEND_FREEZE},     0, 0, FLAGS, .unit = "mode" },
     { "glow",       "", 0, AV_OPT_TYPE_CONST, {.i64=BLEND_GLOW},       0, 0, FLAGS, .unit = "mode" },
--- a/libavfilter/vf_blend_init.h	2026-05-27 20:44:30
+++ b/libavfilter/vf_blend_init.h	2026-05-27 20:45:07
@@ -116,6 +116,7 @@
     case BLEND_DIVIDE:       param->blend = blend_divide_##depth##bit;       break;   \
     case BLEND_DODGE:        param->blend = blend_dodge_##depth##bit;        break;   \
     case BLEND_EXCLUSION:    param->blend = blend_exclusion_##depth##bit;    break;   \
+    case BLEND_INCLUSION:    param->blend = blend_inclusion_##depth##bit;    break;   \
     case BLEND_EXTREMITY:    param->blend = blend_extremity_##depth##bit;    break;   \
     case BLEND_FREEZE:       param->blend = blend_freeze_##depth##bit;       break;   \
     case BLEND_GLOW:         param->blend = blend_glow_##depth##bit;         break;   \
